### DISCUSSION TITLE
simulations run failed:some variables became negative

### DISCUSSION THREAD

**Discussion by xixi-1995** [Open-Systems-Pharmacology/Forum#811] [Questions & Problems] [answered]
Hi, everyone,
I met some problem when I run my simulation, as the picture described, I'm wondering how to solve it, I have tried the first solution mentioned  by picture, but still failed. I really need your help, thanks very much! 
![image](https://user-images.githubusercontent.com/54713645/83592293-a1655a80-a58c-11ea-8e52-0e491bb65b17.png)

**Comment by tobiasK2001** [comment, marked as answer]
Hi,
are you using an oral formulation? Maybe this applies: 
https://github.com/Open-Systems-Pharmacology/Forum/issues/408#issuecomment-546601576

Best, Tobias
